Lib/asyncio/base_events.py

```diff
@@ -1144,7 +1144,7 @@ async def create_connection(
                     (functools.partial(self._connect_sock,
                                        exceptions, addrinfo, laddr_infos)
                      for addrinfo in infos),
-                    happy_eyeballs_delay, loop=self)
+                    happy_eyeballs_delay)
```

Comment:
No need to change this now?

Reply:
I added it back but I don't think it's necessary, loop would be the running loop anyways at this point.